fullstory.go

```diff
@@ -134,5 +134,15 @@ func (c *Client) ExportData(id int) (ExportData, error) {
 		return nil, err
 	}
 
+	if len(modifyReq) > 0 {
+		modifyReq[0](req)
+	}
+
 	return c.doReq(req)
 }
+
+func WithUserAgent(ua string) func(r *http.Request) {
```

Comment:
Would like to not add this API to the `fullstory` package, if that sounds good to you. (Clients can define this individually if they wish.)